Sources/PortalSwift/Provider/PortalProvider.swift

```diff
@@ -532,16 +532,6 @@ public class PortalProvider {
  * Supporting Structs
  **********************************/
 
-/// A list of EVM networks.
```

Comment:
Is this depended on in the example app anywhere? Or by customers anywhere if its Public?

Reply:
Nope. This enum is from 2+ years ago and is mostly useless at this point, since it's just a map of ETH mainnet and various testnets.